scripts/utils/docs-content-tools/handlers.js

```diff
@@ -0,0 +1,142 @@
+const glob = require('glob');
+const frontmatterGH = require('@github-docs/frontmatter');
+const fs = require('fs');
+
+/**
+ * Adds slashes to beginning and end of directory if missing
+ * @param {String} directory - The directory to add slashes to
+ * @returns {String} The directory with slashes added if necessary
+ */
+const formatDirectory = (directory) => {
+  !directory.startsWith('/') && (directory = `/${directory}`);
+  !directory.endsWith('/') && (directory = directory.concat('/'));
+  return directory;
```

Comment:
Kinda nitpicky: would you be willing to put these into `if` / `else` blocks? I think this might be a little easier to read:

```suggestion
  if (!directory.startsWith('/')) {
    return `/${directory}`;
  } else if (!directory.endsWith('/')) {
    return directory.concat('/');
  }

  return directory;
```